launcher/modManager/cmodlistview_moc.cpp

```diff
@@ -352,8 +353,13 @@ void CModListView::on_allModsView_activated(const QModelIndex & index)
 
 void CModListView::on_lineEdit_textChanged(const QString & arg1)
 {
-	QRegExp regExp(arg1, Qt::CaseInsensitive, QRegExp::Wildcard);
-	filterModel->setFilterRegExp(regExp);
+	auto reOptions =  QRegularExpression::CaseInsensitiveOption;
+	auto baseStr = QRegularExpression::wildcardToRegularExpression(arg1);
+	//Hack due to lack QRegularExpression::UnanchoredWildcardConversion in Qt5
+	baseStr.chop(3);
+	baseStr.remove(0,5);
```

Comment:
I suggest to use proper code for Qt 6: wrap it in `#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)`

Reply:
@kambala-decapitator but this code is proper too, and #if QT_VERSION looks ugly. But if you wish - I can add this too.

Reply:
personally I prefer proper code if possible instead of hacks (to me it's not clear at all what is the result before the hack without actually running a test code). Let's wait for another opinion.

Reply:
@kambala-decapitator done.

Reply:
@kambala-decapitator we generate Perl-compatible regex in this constructor. Before the hack - we get string like "\A[?:H]\z", after hack we get H, if we requested only "H" letter in UI.

Reply:
So, I just strip extra anchoring. It is the same for any requested string, I did a couple of checks yesterday.

Reply:
thanks a lot, now it's clear!